src/extension.ts

```diff
@@ -104,9 +104,28 @@ export async function activate(context: vscode.ExtensionContext) {
             open(hoverFile);
         }
     });
-
     context.subscriptions.push(clickInclude);
 
+    // Command for getting page ast
+    const getPageAST: vscode.Disposable = vscode.commands.registerCommand('snooty.getPageAST', async () => {
+        const textDocument: vscode.TextDocument = vscode.window.activeTextEditor.document;
+        const fileName: string = textDocument.fileName;
+        
+        // Only valid on .txt site pages
```

Comment:
We'll want to relax this restriction eventually, but it's probably okay to leave it be for the MVP